packages/tasks/src/model-libraries.ts

```diff
@@ -90,6 +90,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/facebookresearch/audiocraft",
 		snippets: snippets.audiocraft,
 		filter: false,
+		countDownloads: `path:"state_dict.bin" OR path:"config.json"`,
```

Comment:
Nice, this will also enable download stats for repos such as https://huggingface.co/facebook/magnet-small-10secs 

cc @Vaibhavs10